panel/applets/icon-tasklist/WindowListItem.vala

```diff
@@ -0,0 +1,138 @@
+/*
+ * This file is part of budgie-desktop
+ *
+ * Copyright (C) 2016 Fernando Mussel <fernandomussel91@gmail.com>
```

Comment:
Sorry, this was not intentional. The code is new but I wanted to animate the ListBoxRow removal, so I used your code for that. I will be sure to change this and be more attentive to it in the future.